docs/index.bs

```diff
@@ -3175,6 +3235,78 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Return |response|.
   </section>
 
+  <section algorithm>
+    <h3 id="verify-router-rule-algorithm"><dfn>VerifyRouterRule</dfn></h3>
```

Comment:
Let's consider in a follow up PR, but we may be able to cover some points concerned in https://github.com/WICG/service-worker-static-routing-api/issues/14 here. More specifically, throw an error if the router rules contain "fetch-handler" but the SW doesn't have any fetch handers or empty fetch handler flag is true.

Reply:
I do not have a plan to implement fetch-event source in this PR yet.  Therefore, it might not be affected by  WICG/service-worker-static-routing-api#14.  Or, is it better to have that in this PR?

Reply:
Sure.  Thanks for the clarification.